content/articles/creating-and-using-azure-function-serverless-architecture-in-azure-portal-and-visual-studio-2019/index.md

```diff
@@ -0,0 +1,216 @@
+- **TimerTrigger**- Allows for the execution of tasks on a set timetable.
+- **BlobTrigger**- When you add a new blob to your Azure Storage account.
+- **CosmosDBTrigger**- In a NoSQL database, in response to new or changed documents.
+
+Azure Functions are more flexible and easier to manage in general. WebJobs, on the other hand, are a better choice when: - the code has to be maintained as part of an existing App Service application, such as in the same Azure DevOps environment. - You must have total control over the object that listens for events that trigger the execution of the code.
+
+### How to Make an Azure Function using Azure Portal
+
+In this section, we'll go over how to use the Azure Portal to build an Azure function. We must first construct a Function App in Azure Portal before we can create an Azure Function. After that, we can use the Function App to develop various types of Azure Functions.
+
+#### Creating Function App using Azure Portal
+
+**Step 1a**:
+
+To begin, we must first log into the Azure Portal. From the Dashboard, select the Create a Resource option. Then, from the resource list, choose Function App options.
+
+![Step 1a](engineering-education-creating-and-using-azure-function-step-1a.png)
+
+**Step 2a**:
+
+To construct a function app, we must now supply the following choices in the Create Function App section.
+
+- Resource Group — Choose an existing Resource Group or create one from scratch.
+- The name of the function app is contained in this field.
+- Publish - Choose between Code and Docker Container. This option specifies the environment in which we want to run our function app. We'll use Code for our demonstration.
+- Runtime Stack - We must choose which language to construct the Azure Function in. The programming language we choose is.NET.
+- Version-When you choose Runtime Stack from the menu, the most recent version of that programming language is automatically selected in the version dropdown.
+- Region - We must choose the region in which the data centre will be located.
+After entering the values for the aforementioned fields, we must click the Create + Review Button. Then press the Create Button button.
+
+![Step 2a](engineering-education-creating-and-using-azure-function-step-2a.png)
+
+**Step 3a**:
+
+Once the Function App has been deployed, navigate to the function app and it will appear as seen below.
+
+![Step 3a](engineering-education-creating-and-using-azure-function-step-3a.png)
+
+**Step 4a**:
+
+If we copy the Function App URL and paste it into a browser tab, it will show that our Function App is already up and running.
+
+Our Function App is now available. Within this Function App, we must develop our first Azure Function.
+
+#### Creating Azure Function using Azure Portal
+
+**Step 1b**:
+
+Now, under the Function App area, select the Function choices from the left-side menu panel.
+
+![Step 1b](engineering-education-creating-and-using-azure-function-step-1b.png)
+

```

Comment:
Fix all the images links, format is as follows: /engineering-education/folder_name/image_name.